README.md

```diff
@@ -30,10 +39,25 @@ Configuration options are documented in [Configuration.md](https://github.com/gi
 
 Before upgrading, please check [upgrading.md](https://github.com/giantswarm/cert-manager-app/blob/master/docs/upgrading.md).
 
+## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new branch with name `master#release#vX.X.X`. Automation will create a release PR.
+* Merging the release PR will push a new git tag and trigger a new tarball to be pushed to the
+  [giantswarm-catalog].
+* Test and verify the cert-manager release across supported environments in a new or existing WIP platform release.
+
+## Contributing & Reporting Bugs
+If you have suggestions for how `cloudflared` could be improved, or want to report a bug, open an issue! We'd love all and any contributions.
```

Comment:
This should be `cert-manager`, not `cloudflared`, right?

Reply:
missed that one, thanks!